src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -200,19 +201,25 @@ limitations under the License.
 
   <enum name="PresetScenarioEnum" type="enum8">
     <cluster code="0x0201"/>
-    <item name="Unspecified" value="0x00"/>
     <item name="Occupied" value="0x01"/>
     <item name="Unoccupied" value="0x02"/>
     <item name="Sleep" value="0x03"/>
     <item name="Wake" value="0x04"/>
     <item name="Vacation" value="0x05"/>
-    <item name="UserDefined" value="0x06"/>
+    <item name="GoingToSleep" value="0x06"/>
+    <item name="UserDefined" value="0xFE"/>
   </enum>
 
+  <struct name="AtomicAttributeStatusStruct" apiMaturity="provisional">
+    <cluster code="0x0201"/>
```

Comment:
Again, this should end up a global struct and should live in global-structs.xml, right?  Put it there now, and we'll remove the `<cluster>` bit from it once that's supported.

Reply:
OK, relocated to global-structs.xml